realm/src/main/java/io/realm/internal/SharedGroup.java

```diff
@@ -40,9 +40,9 @@
         MEM_ONLY(1);
         //ASYNC(2); // TODO: re-enable when possible
 
-        final int value;
+        public final int value;
```

Comment:
No. My fault. I saw Enum's equals method is declared as final but didn't realize for the Enum comparision `==` is actually good enough. Will change it.